src/System.Collections/pkg/System.Collections.pkgproj

```diff
@@ -8,8 +8,14 @@
     <ProjectReference Include="..\ref\4.0.0\System.Collections.depproj">
```

Comment:
Instead of bumping it manually, I'll just remove the version from here so that it automatically gets calculated. That should give us 4.1.0